src/types.js

```diff
@@ -377,11 +377,13 @@ export type SharedText = {|
 export type SharedImage = {|
   type: 'image',
   sharedImageUrl: string,
+  fileName: string,
 |};
 
 export type SharedFile = {|
   type: 'file',
   sharedFileUrl: string,
+  fileName: string,
 |};
```

Comment:
added the comments,

on a side note this makes me wonder why notification has its own separate types.js (why not include it in the main types?)

Reply:
Thanks!

> on a side note this makes me wonder why notification has its own separate types.js (why not include it in the main types?)

Good question. Two reasons:
* It's generally more helpful to have the code organized primarily by what general subsystem it's part of -- like notifications, or sharing, or the server API -- rather than by what kind of code it is, like is it type definitions. For example when you're working on notifications, you might be touching a bunch of different notifications-related files; but it's less common that you'd be working on "types, in general", and be touching a bunch of different types-related files.
* Having a single global "types" file tends to create big import cycles. Those can be bad for a variety of reasons; they tend to make the codebase harder for both tools and people to work with. Here's an example we ran into: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.23M3745.20Server.20version.20to.20Sentry/near/930319